assets/queries/cloudFormation/security_groups_for_ssh/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "security_groups_for_ssh",
+  "queryName": "Security Groups For SSH",
```

Comment:
Change the query name, it should describe the problem:
"queryName": "Security Group Unrestricted Access To SSH",